voxblox/include/voxblox/core/esdf_map.h

```diff
@@ -30,20 +30,29 @@ class EsdfMap {
     block_size_ = config.esdf_voxel_size * config.esdf_voxels_per_side;
   }
 
-  EsdfMap(const std::string& file_path)
-      : esdf_layer_(io::LoadOrCreateLayerHeader<EsdfVoxel>(file_path,
-                                                           0.2,
-                                                           16u)),
+  explicit EsdfMap(const std::string& file_path)
+      : esdf_layer_(
+            io::LoadOrCreateLayerHeader<EsdfVoxel>(file_path, 0.2, 16u)),
         interpolator_(esdf_layer_.get()) {
-    if (!io::LoadBlocksFromFile<EsdfVoxel>(file_path,
-                                           Layer<EsdfVoxel>::BlockMergingStrategy::kProhibit,
-                                           esdf_layer_.get())) {
+    if (!io::LoadBlocksFromFile<EsdfVoxel>(
+            file_path, Layer<EsdfVoxel>::BlockMergingStrategy::kProhibit,
```

Comment:
can't add a CHECK before the same path is already used in the intializer list, but I added a check inside both LoadBlocksFromFile and LoadOrCreateLayerHeader.